src/main/java/redis/clients/jedis/ScoreValueTuple.java

```diff
@@ -0,0 +1,50 @@
+package redis.clients.jedis;
+
+import redis.clients.jedis.util.SafeEncoder;
+
+import java.util.Arrays;
+
+public class ScoreValueTuple extends Tuple {
```

Comment:
The name `ScoreValueTuple` doesn't feel right. The value score itself is actually contained in Tuple.

The best I can come up with right now is `KeyedTuple`. But fee free to use your imagination.

Reply:
Agree with `KeyedTuple`. Thanks.